Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/TransformToolIntegrationTest.java

```diff
@@ -382,6 +382,7 @@ public void testIfDrawingSurfaceBoundsAreFoundAndNotCropped() {
 				.checkBitmapDimension(initialWidth, initialHeight);
 	}
 
+	@Ignore("Process crashes on Jenkins")
 	@Test
 	public void testIfClickOnCanvasCrops() {
```

Comment:
```suggestion
	@Test
	public void testIfClickOnCanvasCrops() {
```
Why totally ignore it? I mean I see that it sometimes crashes, e.g., 
```
Tests / Device Tests / org.catrobat.paintroid.test.espresso.tools.TransformToolIntegrationTest.testIfClickOnCanvasCrops (from org.catrobat.paintroid.test.espresso.BitmapIntegrationTest)
``` 
and 
```
Stacktrace
Test failed to run to completion. Reason: 'Instrumentation run failed due to 'Process crashed.''. 
```
Still, looking at https://jenkins.catrob.at/job/Paintroid/job/develop/test_results_analyzer/ I see that all recent jenkins runs for the develop branch are green for this test:
![image](https://user-images.githubusercontent.com/1747888/115132116-785f8400-9ffd-11eb-9f6b-0a2025808942.png)
![image](https://user-images.githubusercontent.com/1747888/115132083-477f4f00-9ffd-11eb-9dd9-ff44426e44d2.png)

Same also for recent PRs:
https://jenkins.catrob.at/job/Paintroid/view/change-requests/job/PR-870/test_results_analyzer/
![image](https://user-images.githubusercontent.com/1747888/115138686-e623a400-a02d-11eb-9467-8243e8b0bcce.png)

I therefore suggest not skipping this test. Is there a better way, i.e., fixing the crash? Or can't we make it flaky? Or is there a problem because of the crash?